src/runtime/runtime_stm32l4x6.go

```diff
@@ -0,0 +1,29 @@
+//go:build stm32 && stm32l4x6
+
+package runtime
+
+import (
+	"device/stm32"
+)
+
+/*
+clock settings
+
+	+-------------+-----------+
+	| LSE         | 32.768khz |
+	| SYSCLK      | 80mhz    |
+	| HCLK        | 80mhz    |
+	| APB1(PCLK1) | 80mhz    |
+	| APB2(PCLK2) | 80mhz    |
+	+-------------+-----------+
+*/
+const (
+	HSE_STARTUP_TIMEOUT = 0x0500
+	PLL_M               = 1
+	PLL_N               = 60
+	PLL_P               = RCC_PLLP_DIV2
```

Comment:
not sure if any of these `PLL_*` constants are correct

Reply:
fixed the issue by copying the values in L4x2. Good following up reading: https://community.st.com/t5/stm32-mcus/part-1-introduction-to-the-stm32-microcontroller-clock-system/ta-p/605369